code/modules/uplink/uplink_items/uplink_dangerous.dm

```diff
@@ -204,7 +204,7 @@
 	desc = "Though capable of near sorcerous feats via use of hardlight holograms and nanomachines, they require an \
 			organic host as a home base and source of fuel. Holoparasites come in various types and share damage with their host."
 	item = /obj/item/storage/box/syndie_kit/nukieguardian
-	cost = 15
+	cost = 8
```

Comment:
"and operatives it had cost the same amount, and now 12."
🤔

Reply:
HAD cost, yes

Reply:
I am against giving this to ops for so cheap because it's a really powerful tool
support stands (which everyone uses anyways) let the op teleport back to the ship on demand and can heal for free out of crit and while out of combat
the one and only reason I am against this is because support is such a powerful tool and letting nukies have it would basically make it a must buy during warops when there is more than likely enough ghosts to actually accept it
that and chaos and charger make excellent bullet shields with their exceptionally high burn and brute damage respectively plus chaos' innate ability to make you immune to fire ticks